examples/p4est_2d_dgsem/elixir_euler_density_wave_tracers.jl

```diff
@@ -0,0 +1,55 @@
+using OrdinaryDiffEqSSPRK, OrdinaryDiffEqLowStorageRK
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+gamma = 1.4
+flow_equations = CompressibleEulerEquations2D(gamma)
+equations = PassiveTracerEquations(flow_equations, 2)
+
+initial_condition = initial_condition_density_wave
+
+volume_flux = FluxTracerEquationsCentral(flux_ranocha)
+solver = DGSEM(polydeg = 5, surface_flux = FluxTracerEquationsCentral(flux_ranocha))
```

Comment:
```suggestion
solver = DGSEM(polydeg = 5, surface_flux = FluxTracerEquationsCentral(flux_ranocha))
```

The `volume_flux` is never used, is it?